<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>843</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Model2SAS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1147</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport_lattice_model">
     <property name="title">
      <string>Export lattice model</string>
     </property>
     <addaction name="action_exportTxtFile"/>
     <addaction name="action_exportPDBFile"/>
    </widget>
    <addaction name="action_newProject"/>
    <addaction name="action_loadProject"/>
    <addaction name="separator"/>
    <addaction name="action_saveProject"/>
    <addaction name="action_saveSasCurve"/>
    <addaction name="menuExport_lattice_model"/>
    <addaction name="separator"/>
    <addaction name="action_importModels"/>
    <addaction name="action_deleteAllModels"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="action_showAllModels"/>
    <addaction name="action_showSelectedModels"/>
    <addaction name="action_showLatticeModel"/>
    <addaction name="action_showSasCurve"/>
    <addaction name="separator"/>
    <addaction name="action_Cascade"/>
    <addaction name="action_Tile"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="action_configureGpu"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_mathModelGeneration"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindows"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_explorer">
   <property name="minimumSize">
    <size>
     <width>275</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Model Explorer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButton_modifyModel">
       <property name="text">
        <string>modify model</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_showSelectedModels">
       <property name="text">
        <string>show selected model(s)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_importModels">
       <property name="text">
        <string>import model(s)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_showAllModels">
       <property name="text">
        <string>show all models</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_deleteModels">
       <property name="text">
        <string>delete selected model(s)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_projectName">
       <property name="text">
        <string>Project: New Project</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" rowspan="2" colspan="2">
      <widget class="QTableView" name="tableView_models"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_console">
   <property name="windowTitle">
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="textEdit_console">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_controlPanel">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>377</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>400</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Control Panel</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_genPoints">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Generate lattice model</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEdit_interval">
          <property name="text">
           <string>1.0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pushButton_genLatticeModel">
          <property name="text">
           <string>generate</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="lineEdit_gridPointsNum">
          <property name="text">
           <string>10000</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="radioButton_gridPointsNum">
          <property name="text">
           <string>grid points number</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="radioButton_interval">
          <property name="text">
           <string>interval</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_calcSas">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Calculate SAS data</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>q number</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>q_min, q_max</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="lineEdit_qmin">
          <property name="text">
           <string>0.001</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEdit_lmax">
          <property name="text">
           <string>50</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="lineEdit_qmax">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="8" column="3">
         <widget class="QPushButton" name="pushButton_calcSas">
          <property name="text">
           <string>calculate</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLineEdit" name="lineEdit_qnum">
          <property name="text">
           <string>200</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>l_max</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QCheckBox" name="checkBox_useGpu">
          <property name="text">
           <string>use GPU</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Load Project</string>
   </property>
  </action>
  <action name="action_saveProject">
   <property name="text">
    <string>Save project</string>
   </property>
  </action>
  <action name="action_importModels">
   <property name="text">
    <string>Import model(s)</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="action_Cascade">
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="action_Tile">
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="actionControl_Panel">
   <property name="text">
    <string>Contol Panel</string>
   </property>
  </action>
  <action name="action_newProject">
   <property name="text">
    <string>New project</string>
   </property>
  </action>
  <action name="action_deleteAllModels">
   <property name="text">
    <string>Delete all models</string>
   </property>
  </action>
  <action name="action_saveLatticeModel">
   <property name="text">
    <string>Save lattice model</string>
   </property>
  </action>
  <action name="action_saveSasCurve">
   <property name="text">
    <string>Save SAS curve</string>
   </property>
  </action>
  <action name="action_loadProject">
   <property name="text">
    <string>Load project</string>
   </property>
  </action>
  <action name="action_configureGpu">
   <property name="text">
    <string>Configure GPU with PyTorch</string>
   </property>
  </action>
  <action name="action_showLatticeModel">
   <property name="text">
    <string>Show lattice model</string>
   </property>
  </action>
  <action name="action_showSasCurve">
   <property name="text">
    <string>Show SAS curve</string>
   </property>
  </action>
  <action name="action_showAllModels">
   <property name="text">
    <string>Show all models</string>
   </property>
  </action>
  <action name="action_showSelectedModels">
   <property name="text">
    <string>Show selected model(s)</string>
   </property>
  </action>
  <action name="action_mathModelGeneration">
   <property name="text">
    <string>Generate math model</string>
   </property>
  </action>
  <action name="action_exportTxtFile">
   <property name="text">
    <string>txt file</string>
   </property>
  </action>
  <action name="action_exportPDBFile">
   <property name="text">
    <string>PDB file</string>
   </property>
  </action>
  <zorder>dockWidget_controlPanel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
